tango_simlib/tango_launcher.py

```diff
@@ -42,26 +44,30 @@ def register_device(name, device_class, server_name, instance):
     dev_info = tango.DbDevInfo()
     dev_info.name = name
     dev_info._class = device_class
-    dev_info.server = "{}/{}".format(server_name.split('.')[0], instance)
+    dev_info.server = "{}/{}".format(server_name, instance)
```

Comment:
Why was the `split` removed?  The server name on the command line could have an extension.  I think that is why this split used to be here.  E.g. see line 82 in this file.

Reply:
This was removed by error whilst moving code snippets around. Thanks for pointing this out. 
I haven't used/seen server names with extensions yet. I didn't know they could come with names like this. Another thing learnt.